modules/admin_manual/pages/configuration/server/security/hsmdaemon/index.adoc

```diff
@@ -78,21 +70,18 @@ If you are using a different operating system or distribution, please adjust the
 
 ==== Install Using a Preconfigured PKCS11 Module
 
-At least one PKCS11 library is necessary. 
-This is typically provided by an HSM vendor. 
-If a PKCS11 library is not available, you can xref:configuration/server/security/hsmdaemon/softhsm2.adoc[use the software HSM, _SoftHSM2_].
+At least one PKCS11 library is necessary. This is typically provided by an HSM vendor. If a PKCS11 library is not available, you can xref:configuration/server/security/hsmdaemon/softhsm2.adoc[use the software HSM - _SoftHSM2_].
 
 ==== Initialise the Token
 
 Now we can initialize the token:
 
 [source,console]
 ----
-softhsm2-util --init-token --slot 0 --label "My token 1"
+sudo softhsm2-util --init-token --slot 0 --label "My token 1"
 ----
 
-It will ask for two PINs, an SO and a User pin.
-See https://www.opendnssec.org/softhsm/, for more information.
+It will ask for two PINs, an SO and a User pin. See {opendnssec-url}[opendnssec], for more information. The SO PIN can e.g. be used to re-initialize the token and the user PIN is handed out to the application so it can interact with the token.
```

Comment:
```suggestion
It will ask for two PINs, an SO and a User pin. See {opendnssec-url}[opendnssec] for more information. The SO PIN can e.g. be used to re-initialize the token and the user PIN is handed out to the application so it can interact with the token.
```